docs/source/scanning.rst

```diff
@@ -0,0 +1,11 @@
+Scanning
+--------
+Quipucords can scan several types of sources from network systems to vCenter Servers and Satellite servers.
+
+Scanning Lifecycle
+^^^^^^^^^^^^^^^^^^
+A user defines a scan, see `Working with Scans <working_with_scans.html>`_, and triggers scanning to begin. When the user triggers scanning to begin a scan job is create. When a scan job is created it is queued for processing. Due to the light-weight nature of the Quipucords server scan jobs are run serially. Scan jobs will transition from queued to started and processing will begin. A user can select to pause or cancel a scan job that is pending or running. A paused scan job can be restarted, a canceled scan job cannot. A scan job can halt due to a fatal error resulting in a *failed* status, the status message should help determine the cause of the failure. When a scan results in a *completed* status it produces results that can be viewed in an associated report.
```

Comment:
When the user triggers scanning to begin a scan job is create ---> "" created.

Reply:
done